pkg/appliance/functions.go

```diff
@@ -596,6 +584,134 @@ func applyApplianceFilter(appliances []openapi.Appliance, filter map[string]stri
 	return filteredAppliances, nil
 }
 
+func orderAppliances(appliances []openapi.Appliance, orderBy []string, descending bool) ([]openapi.Appliance, error) {
+	// reverse loop the slice to prioritize the ordering. First entered has priority
+	for i := len(orderBy) - 1; i >= 0; i-- {
+		switch orderBy[i] {
+		case "name":
```

Comment:
just and idea, but if these key values in the switch case was a exported map instead, it would be trivial to later add support for tab completion in the command that can use these.

Reply:
a map is not ordered though, so it would be impossible to implement prioritized ordering by using a map.